test/cpp/api/functional.cpp

```diff
@@ -360,3 +360,14 @@ TEST_F(FunctionalTest, LogSigmoid) {
   auto y_exp = torch::log(torch::ones_like(x)/(torch::ones_like(x) + torch::exp(torch::neg(x))));
   ASSERT_TRUE(torch::allclose(y, y_exp, 1e-4, 1e-7));
 }
+
+TEST_F(FunctionalTest, Softmax) {
+  auto input = torch::arange(10, torch::kFloat).reshape({2, 5});
+  auto output = F::softmax(input, SoftmaxOptions(1));
```

Comment:
We can change this to
```suggestion
  auto output = F::softmax(input, /*dim=*/1);
```
to also test `SoftmaxOptions`'s implicit constructor. ;)

Reply:
This is also fixed as well.